wagtail/admin/templates/wagtailadmin/shared/page_status_tag.html

```diff
@@ -1,5 +1,11 @@
+{% load i18n %}
 {% if page.live %}
-    <a href="{{ page.url }}" target="_blank" rel="noopener noreferrer" class="status-tag primary">{{ page.status_string }}</a>
+    <a href="{{ page.url }}" target="_blank" rel="noopener noreferrer" class="status-tag primary" title="Visit the live page">
+        <span class="visuallyhidden">{% trans "Current page status:" %}</span>
+        <span class="">{{ page.status_string }}</span>
+    </a>
 {% else %}
-    <span class="status-tag">{{ page.status_string }}</span>
+    <span class="status-tag">
+        <span class="visuallyhidden">{% trans "Current page status:" %}</span>{{ page.status_string }}
+    </span>
```

Comment:
👍 here I see that `status_string` is already translated when inserted here, so I don’t think the rest of my comments above apply.